src/App.js

```diff
@@ -47,11 +48,13 @@ class App extends Component {
                   path="/login"
                   exact
                   component={Login}/>
+              <Route
+                  path="/businessDetails/:id"
+                  exact
+                  component={BusinessDetails}/>
             </div>
-          </Router>
```

Comment:
Can you add the </Router> and the </div> back in? It's needed for it to compile. I think it got deleted in the merge.